suites/nautilus/rbd/rbd_mirror_using_ceph_ansible.yaml

```diff
@@ -0,0 +1,116 @@
+#===============================================================================================
+# Tier-level: 1
+# Test-Scenario: Configure RBD Mirror using ceph-ansible and run IOs
+#
+# Cluster Configuration:
+#    conf/nautilus/rbd/rbd_mirror_using_ceph_ansible.yaml
+#    No of Clusters : 2
+#    Each cluster configuration
+#    6-Node cluster(RHEL-8.3 and above)
+#    3 MONS, 2 MGR, 3 OSD and 1 RBD MIRROR nodes
+#     Node1 - Mon, Mgr, Installer
+#     Node2 - client, rbd-mirror
+#     Node3 - Mon, OSD, mgr
+#     Node4 - Mon, OSD
+#     Node5 - OSD
+#
+# Sequence of tests -
+#   1) Configure two clusters
+#   2) Handle same cluster names (required for ceph-ansible usage)
+#   3) Add rbd mirror daemon to both clusters
+#   4) Run IOs
+#===============================================================================================
+tests:
+  - test:
+      name: install ceph pre-requisites
+      module: install_prereq.py
+      abort-on-fail: True
+
+  - test:
+      name: ceph ansible
+      module: test_ansible.py
+      clusters:
+        ceph-rbd1:
+          config:
+            ansi_config:
+              ceph_origin: distro
+              osd_auto_discovery: False
+              ceph_stable_rh_storage: True
+              fetch_directory: ~/fetch
+              copy_admin_key: true
+              dashboard_enabled: False
+              osd_scenario: lvm
+              ceph_conf_overrides:
+                global:
+                  osd_pool_default_pg_num: 64
+                  osd_pool_default_pgp_num: 64
+                  mon_max_pg_per_osd: 1024
+                mon:
+                  mon_allow_pool_delete: true
+        ceph-rbd2:
+          config:
+            ansi_config:
+              ceph_origin: distro
+              osd_auto_discovery: False
+              fetch_directory: ~/fetch
+              copy_admin_key: true
+              dashboard_enabled: False
+              osd_scenario: lvm
+              ceph_conf_overrides:
+                global:
+                  osd_pool_default_pg_num: 64
+                  osd_pool_default_pgp_num: 64
+                  mon_max_pg_per_osd: 1024
+                mon:
+                  mon_allow_pool_delete: true
+      desc: setup dual cluster using ceph-ansible
+      destroy-cluster: false
```

Comment:
Can we add a polarion ID?

Reply:
Ref: https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83573366

Reply:
@Ameenasuhani  we are just configuring two clusters here in this test and updating CEPH-83573366 wouldn't make sense
Check two tasks below 81 onwards, where we are actually configuring rbdmirror daemon initiating playbook again, this TC suites there to be updated.

Reply:
@Ameenasuhani Can you please let me know your thoughts

Reply:
@Ameenasuhani gentle reminder

Reply:
@Ameenasuhani a gentle reminder

Reply:
@VasishtaShastry In that case you might want to create a new polarion tc and attach it or modify the existing one